0.X/docker-entrypoint.sh

```diff
@@ -75,7 +75,7 @@ elif consul --help "$1" 2>&1 | grep -q "consul $1"; then
 fi
 
 # If we are running Consul, make sure it executes as the proper user.
-if [ "$1" = 'consul' ]; then
+if [ "$1" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT+x}" ]; then
```

Comment:
```suggestion
if [ "$1" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT:-}" ]; then
```
This shell syntax should also work in the container, but allow the more familiar use of a non-empty value to imply activation of the feature, and empty and unset values implying deactivation of the feature:

```
$ docker run --rm -it alpine:3.9
/ # X=consul # faking positional arg

/ # if [ "$X" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT:-}" ]; then echo a; else echo b; fi
a

/ # CONSUL_DISABLE_PERM_MGMT=1
/ # if [ "$X" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT:-}" ]; then echo a; else echo b; fi
b

/ # CONSUL_DISABLE_PERM_MGMT=
/ # if [ "$X" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT:-}" ]; then echo a; else echo b; fi
a

/ # unset CONSUL_DISABLE_PERM_MGMT
/ # if [ "$X" = 'consul' -a -z "${CONSUL_DISABLE_PERM_MGMT:-}" ]; then echo a; else echo b; fi
a
```

Reply:
Under normal circumstances I would have done it so that activation required the value being `1`, `t`, `true`, `y`, or `yes` and anything else would be assumed to be deactivated.

However the reason for this weird syntax was in keeping with the style of the rest of the entrypoint script. The same `:+x` syntax was used a to enable running setcap on consul and we assumed the non-nullness of the env var implied activation.

We certainly could have this one be different but then it wouldn't be consistent with the others.

Reply:
Ah well in that case I retract the suggestion.